test/acceptance/ApiSubscriptionFieldsUnhappySpec.scala

```diff
@@ -34,10 +34,10 @@ class ApiSubscriptionFieldsUnhappySpec extends AcceptanceTestSpec
   feature("Subscription-Fields") {
     Logger.logger.info(s"App.mode = ${app.mode.toString}")
 
-    scenario("the API is called to GET an unknown subscription identifier") {
+    scenario("the API is called to GET an unknown subscription field") {
```

Comment:
`the API is called to GET non-existing subscription fields`

Reply:
Done